### PR TITLE
#RI-5856 - handle yaml error

### PR DESCRIPTION
**Pull request by AmirAllayarovSofteq**

### CODE REVIEW

File: redisinsight/ui/src/utils/apiResponse.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { AxiosError } from 'axios'
-import { first, isArray, get } from 'lodash'
-import { AddRedisDatabaseStatus, EnhancedAxiosError, IBulkOperationResult } from 'uiSrc/slices/interfaces'
+import {AxiosError, AxiosResponse} from 'axios'
+import {first, isArray, get, set} from 'lodash'
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/utils/apiResponse.ts:2]: please, add spaces

File: redisinsight/ui/src/utils/transformers/transformRdiPipeline.ts [hunk 2]
```diff
@@ -9,15 +27,40 @@ export const pipelineToYaml = (pipeline: IPipelineJSON) => ({
...
+  }
+  const errors = []
+
+  try {
+    result.config = yaml.load(config) || {}
+  } catch (e) {
+    if (e instanceof YAMLException) {
+      errors.push({ filename: 'config', msg: e.reason })
+    }
+  }
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/utils/transformers/transformRdiPipeline.ts:43]: could you please use here yamlToJson function

File: redisinsight/ui/src/pages/rdi/pipeline-management/components/jobs-panel/Panel.tsx [hunk 3]
```diff
@@ -111,7 +110,7 @@ const DryRunJobPanel = (props: Props) => {
         id: rdiInstanceId,
       },
     })
-    dispatch(rdiDryRunJob(rdiInstanceId, JSON.parse(input), yaml.load(job)))
+    dispatch(rdiDryRunJob(rdiInstanceId, JSON.parse(input), job))
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/pages/rdi/pipeline-management/components/jobs-panel/Panel.tsx:113]: can we fail here with JSON.parse?

Comment (AmirAllayarovSofteq) [reply redisinsight/ui/src/pages/rdi/pipeline-management/components/jobs-panel/Panel.tsx:113]: Nope, we validate submit button. If JSON.parse failed - the button will be disabled

File: redisinsight/ui/src/utils/transformers/transformRdiPipeline.ts (hunk 2 above)
Comment (AmirAllayarovSofteq) [reply redisinsight/ui/src/utils/transformers/transformRdiPipeline.ts:43]: Good point. Thank you.

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**